src/dune_pkg/fetch.ml

```diff
@@ -56,11 +56,15 @@ module Curl = struct
            ([ Pp.textf "curl returned an invalid error code %d" exit_code ] @ stderr)))
     else (
       Path.unlink_no_err stderr;
-      match Int.of_string http_code with
+      (* Since we had to quote the return http_code, we need to strip it *)
+      match
+        Int.of_string (String.sub http_code ~pos:1 ~len:(String.length http_code - 2))
+      with
       | None ->
         Error
           (User_message.make
              [ Pp.textf "curl returned an HTTP code we don't understand: %S" http_code ])
+      | Some 200 -> Ok ()
```

Comment:
This is redundant.

Reply:
I meant to also change the line following, let me do that.

Reply:
Please don't add pointless cosmetic changes to this PR.

Reply:
OK removing.